### DISCUSSION TITLE
changing texstudio_de.ts allowed?

### DISCUSSION THREAD

**Discussion by octaeder** [texstudio-org/texstudio#2725] [Q&A] [answered]
Is it correct/possible to change a translation file if I want to correct a text? Example (this translation is actually wrong, that's why I ask):

    <message>
        <location filename="../src/configdialog.ui" line="2293"/>
        <source>Overwrite Closing Bracket Following a Placeholder</source>
        <translation>Überschreibe schließende Klammer, die von einem Platzhalter gefolgt wird</translation>
    </message>

With my approach I would change the text in the translation text and make a PR. Is this a valid procedure?

Thx

**Comment by sunderme** [comment, marked as answer]
no, use https://www.transifex.com/texstudio/texstudio

**Comment by sunderme** [comment]
see also webpage (texstudio.org)

**Comment by octaeder** [comment]
what are next steps after saving texts on transifex?
will I see changes in github?
is it possible to test changes?

**Comment by sunderme** [comment]
I update the translations regularly on github repository, so they are already here.
You can download texstudio_de.ts from transifex to test.

**Comment by octaeder** [comment]
sorry for so many questions. What's the difference between the two? I work on the first:

![grafik](https://user-images.githubusercontent.com/102688820/206004753-bf05f14b-de87-4d71-8304-3e1eedfc3b6d.png)

**Comment by sunderme** [comment]
txs only uses the first one. 
In theory the second would allow for regional differences, but it makes not much sense. (for German)

**Comment by octaeder** [comment]
I downloaded the file `for_translation_texstudio_texstudio_ents_de.ts`, renamed and moved it to the translation folder of the build directory. Then I started txs. But changes done 1h ago are missing, even so the new file shows the changes done. What do I have to do?

**Reply by sunderme** [reply]
https://doc.qt.io/qt-6/linguist-manager.html#using-lrelease
